.github/workflows/web-example-ghpages.yml

```
@@ -1,16 +1,28 @@
 name: Flutter Web Example GH-Pages
 
+# on:
+#   release:
+#     types: [published]
 on:
-  release:
-    types: [published]
+  pull_request:
 
 jobs:
   build:
     runs-on: ubuntu-latest
 
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+
     steps:
       - uses: actions/checkout@v2
       - uses: subosito/flutter-action@v1
-      - uses: erickzanardo/flutter-gh-pages@v3
+      - uses: erickzanardo/flutter-gh-pages@v6
```

We need to ping this (given it's not a verified publisher)
```suggestion
      - uses: erickzanardo/flutter-gh-pages@66e3e7b317d4fef824b2cfd3c9a4a112d321f33d
```